modules/node_groups/outputs.tf

```
@@ -0,0 +1,25 @@
+output "node_groups" {
+  description = "Outputs from EKS node groups. Map of maps, keyed by var.node_groups keys"
+  value = {
```

Keep it simple, keep it versatile:
```suggestion
  value = aws_eks_node_group.workers
```